packages/eslint-plugin-pf-codemods/lib/rules/table-removed-transforms.js

```diff
@@ -3,27 +3,55 @@ const { getPackageImports } = require('../helpers');
 // https://github.com/patternfly/patternfly-react/pull/4170
 module.exports = {
   create: function(context) {
-    const imports = getPackageImports(context, '@patternfly/react-table')
-      .filter(specifier => specifier.imported.name === 'cellWidth');
+    const tableImports = getPackageImports(context, '@patternfly/react-table');
     
-    return imports.length === 0 ? {} : {
-      CallExpression(node) {
-        const maxNode = node.callee.name && 
-          node.callee.name === 'cellWidth' &&
-          node.arguments.find(arg => arg.value === 'max');
-        if (!maxNode) {
-          return;
-        }
-        const nodeText = context.getSourceCode().getText(maxNode);
-        context.report({
-          node, 
-          message: `cellWidth(${nodeText}) has been replaced with cellWidth(100)`, 
-          fix(fixer) {
-            return fixer.replaceText(maxNode, '100');
+    if (tableImports.length) {
```

Comment:
Don't need this if statement. I'm not sure what exactly happens when we return `undefined` rather than `{}` from this method... but it can't be good!